doc/how-to/seed_cas2_data_in_non_production_environments.md

```diff
@@ -0,0 +1,50 @@
+# How to seed CAS2 data in non-production environments
+
+The seeding of `Cas2Application` and `Cas2Assessment` data is mostly done automatically for developers when the API
+server is started up on the local, dev and test environments, creating a number of Applications and Assessments that can
+be used to test out the API or UI.
+
+Due to some confusion between the two ways of seeding data, this how-to aims to clarify the use case for each type of
+seeding.
+
+## Seed jobs
```

Comment:
For applications there's that gotcha around seeding also setting `created_at` so checking your DB directly to ensure seeds are run is not the way to test it.

Reply:
Yeah, might be useful to prevent people getting confused like we did! Have added.